test/scripts/products/h24-verify-selfmanaged-apicast-and-custom-policy/test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#  Prerequisites:
+# 		oc logged as cluster admin (kubeadmin)
+#  Recommended, before running current version of script, to have following:
+# 		Password for customer, if testing-idp is not yet available.
+# 		Customer Token, if testing-idp and customers are already available.
+# 		Open 3scale Admin Portal, to be ready to put updates. Details will be notified by script.
+
+
+go run h24-verify-selfmanaged-apicast-and-custom-policy.go \
+-3scale-version="2.11" \
+-apicast-operator-version="v0.5.0" \
+-apicast-image-stream-tag="3scale2.11.0" \
+-apicast-namespace="selfmanaged-apicast" \
+-create-testing-idp=false \
+-use-customer-admin-user=true \
+-promote-manually=true
```

Comment:
could we expose these values in case we want pass new values in future release where the version has moved on for example..

Reply:
yes, sure, this file exactly for this, as for example, I started work with apiCast 0.4.0, but later switched to newer version. Same for 3scale, I worked before with 2.10, and later switched to 2.11. The parameters could be also entered from command line, suppose this will work this way when we will integrate the script into Pipeline; parameters will be entered from Jenkins.

Reply:
Done, 3scale version retrieved in code - string(integreatlyv1alpha1.Version3Scale)